# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/x11/mozilla/home-bookmarks.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./profile/defaults/bookmarks.html.orig	Thu Dec 20 22:23:08 2001
+++ ./profile/defaults/bookmarks.html	Thu Dec 20 22:30:17 2001
@@ -6,6 +6,13 @@
 <TITLE>Bookmarks</TITLE>
 <H1>Bookmarks</H1>
 
+<DT><H3 ADD_DATE="1004175168" LAST_MODIFIED="1004175272" ID="NC:BookmarksRoot#$2a05883c">ROCK Linux</H3>
+ <DL><p>
+<DT><A HREF="http://www.rocklinux.org/" ADD_DATE="0" LAST_VISIT="0" LAST_MODIFIED="0">ROCK Linux Homepage</A>
+<DT><A HREF="http://www.rocklinux.net/" ADD_DATE="0" LAST_VISIT="0" LAST_MODIFIED="0">ROCK Linux Web Portal</A>
+<DT><A HREF="http://www.rocklinux.com/" ADD_DATE="0" LAST_VISIT="0" LAST_MODIFIED="0">ROCK Linux Shop</A>
+</DL><p>
+
 <DL><p>
     <DT><H3 ADD_DATE="961112773" LAST_MODIFIED="0" PERSONAL_TOOLBAR_FOLDER="true" ID="NC:PersonalToolbarFolder">Personal Toolbar Folder</H3>
     <DL><p>
--- ./xpfe/browser/resources/locale/en-US/region.properties.orig	Sat May 25 05:01:32 2002
+++ ./xpfe/browser/resources/locale/en-US/region.properties	Sat May 25 05:04:23 2002
@@ -1,5 +1,5 @@
 # navigator.properties
-homePageDefault=http://www.mozilla.org/
+homePageDefault=http://www.rocklinux.net/
 shopKeyword=keyword:shop [Product]
 quoteKeyword=keyword:quote [Enter symbol here]
 localKeyword=keyword:zip [Your zip code]
@@ -11,7 +11,7 @@
 #
 # all.js
 #
-browser.startup.homepage=http://www.mozilla.org/start/
+browser.startup.homepage=http://www.rocklinux.net/
 browser.throbber.url=http://www.mozilla.org/
 browser.search.defaulturl=http://search.netscape.com/cgi-bin/search?search=
 
@@ -20,4 +20,4 @@
 
 #config.js
 #
-startup.homepage_override_url=http://www.mozilla.org/start/
+startup.homepage_override_url=http://www.rocklinux.net/
